### DISCUSSION TITLE
夸克盘，用群晖Cloudsync同步，文件一直不停报错，重复上传，实际上网盘上已经有收到很多份

### DISCUSSION THREAD

**Discussion by Richar-Song** [AlistGo/alist#1997] [General]
Alist docker安装之前用V2.6有问题，今天升级到V3.2版本问题依然。

<img width="658" alt="image" src="https://user-images.githubusercontent.com/106940548/195493848-a7abcd0b-36b4-4ab4-8caf-f37e69754eb9.png">
<img width="650" alt="image" src="https://user-images.githubusercontent.com/106940548/195494047-92353712-34a8-42d4-80b0-66815af5633c.png">

**Comment by shaojs321** [comment]
我部署在heroku也遇到这问题，用群晖Cloudsync同步的时候经常重复上传，换了vps  docker部署就没这问题

**Comment by Richar-Song** [comment]
经过仔细观察和试验，终于找到问题了，原来是夸克网盘不支持文件名中含有 ' 这个单引号，很多英文歌曲或者英文资料文件名都包含这个。上传到夸克网盘后网盘自动将 ’  替换成 &#39,然后cloudsync在远端没找到，然后就一直重复上传，弱智的夸克不停的收，改名，越攒越多。

**Reply by shaojs321** [reply]
> 经过仔细观察和试验，终于找到问题了，原来是夸克网盘不支持文件名中含有 ' 这个单引号，很多英文歌曲或者英文资料文件名都包含这个。上传到夸克网盘后网盘自动将 ’ 替换成 &#39,然后cloudsync在远端没找到，然后就一直重复上传，弱智的夸克不停的收，改名，越攒越多。

明天去试试

**Reply by shaojs321** [reply]
> > 经过仔细观察和试验，终于找到问题了，原来是夸克网盘不支持文件名中含有 ' 这个单引号，很多英文歌曲或者英文资料文件名都包含这个。上传到夸克网盘后网盘自动将 ’ 替换成 &#39,然后cloudsync在远端没找到，然后就一直重复上传，弱智的夸克不停的收，改名，越攒越多。
> 
> 明天去试试

刚看了一下我的文件，应该和你不是同一个问题，我换vps部署就没有重复文件。是我之前用heroku部署的原因

**Comment by shaojs321** [comment]
话说，夸克同步的时候必须走代理，你是走的vps吗，流量吃得消？哪家啊？  我目前是部署在openwrt上，有很多视频文件，但一直走硬盘我怕迟早要废

**Reply by Richar-Song** [reply]
是下载要走代理吧？我是部署在本地群晖里，夸克盘只做云备份盘，基本不下载。
